src/sisl/viz/figure/matplotlib.py

```diff
@@ -380,6 +381,8 @@ def draw_multisize_line(
         # Set the values used for colormapping
         lc.set_linewidth(line.get("width", 1))
         lc.set_linestyle(self._plotly_dash_to_matplotlib(line.get("dash", "solid")))
+        lc.set_color(line.get("color"))
+        lc.set_alpha(line.get("opacity"))
```

Comment:
a bit pedantic here, but matplotlib users are used to using `alpha=`, could it allow *both* things?

Reply:
I think it would become a complete chaos if I try to support all keywords of all backends :sweat_smile: 
I took the decision to stick to plotly terminology in general to make things easier to maintain and document.

Reply:
ok, fair...